src/main/java/org/usfirst/frc4904/robot/commands/controlpanel/SpinPanelThreeTimes.java

```diff
@@ -0,0 +1,63 @@
+package org.usfirst.frc4904.robot.commands.controlpanel;
+
+import edu.wpi.first.wpilibj2.command.SequentialCommandGroup;
+
+import org.usfirst.frc4904.standard.commands.WaitUntil;
+import org.usfirst.frc4904.standard.subsystems.motor.Motor;
+
+import java.util.function.Supplier;
+
+import com.ctre.phoenix.sensors.CANCoder;
+
+import org.usfirst.frc4904.robot.commands.controlpanel.StartPanelMotor;
+import org.usfirst.frc4904.robot.commands.controlpanel.StopPanelMotor;
+
+public class SpinPanelThreeTimes extends SequentialCommandGroup {
+    private final double SPEED = -1d;
+    private final double CONTROL_PANEL_DIAMETER = -1d;
+    private final double PANEL_SPINNER_DIAMETER = -1d;
+    private final double PANEL_SPINNER_RATIO = CONTROL_PANEL_DIAMETER / PANEL_SPINNER_DIAMETER;
+    private final double DEGREES_IN_A_CIRCLE = 360d;
```

Comment:
I would personally not make a constant out of the number 360, but it's up to you.